File: mobile/src/main/java/tr/xip/prayertimes/api/objects/County.java
package tr.xip.prayertimes.api.objects;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class County implements Serializable {
    private int databaseId;

    @SerializedName("IlceId")
    private String id;

    @SerializedName("IlceAdi")
    private String name;

    @SerializedName("error")
    private String error;

    public County(int databaseId, String id, String name) {
        this.databaseId = databaseId;
        this.id = id;
        this.name = name;
    }

    public County(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getError() {
        return error;
    }
}

File: mobile/src/main/kotlin/tr/xip/prayertimes/model/County.kt
package tr.xip.prayertimes.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class County : Serializable {
    var databaseId: Int? = null
        internal set

    @SerializedName("IlceId")
    var id: String? = null

    @SerializedName("IlceAdi")
    var name: String? = null

    @SerializedName("error")
    val error: String? = null

    constructor(databaseId: Int, id: String, name: String) {
        this.databaseId = databaseId
        this.id = id
        this.name = name
    }

    constructor(id: String, name: String) {
        this.id = id
        this.name = name
    }
}
